rosetta-lang/src/main/java/com/regnosys/rosetta/utils/DeepFeatureCallUtil.java

```diff
@@ -30,7 +30,7 @@ public Map<String, RAttribute> findDeepFeatureMap(RDataType type) {
 			result.put(attr.getName(), attr);
 		}
 		for (RAttribute attr : allNonOverridenAttributes) {
-			RType attrType = attr.getRType();
+			RType attrType = attr.getRMetaAnnotatedType().getRType();
```

Comment:
This one might also need to include metadata

Reply:
now redundant closing